runtime/ui/src/sys/linux/cuivectorfont.cpp

```diff
@@ -29,66 +31,556 @@ define_holder(ILTTexInterface, pTexInterface);
 static ILTClient *ilt_client;
 define_holder(ILTClient, ilt_client);
 
+#define WIDTHBYTES( nBitCount, nPixelWidth ) (((( uint32 )nBitCount * nPixelWidth + 7) / 8 + 3) & ~3 )
```

Comment:
does this need to be a macro?
could this be an inline static or constexpr function